python-interface/src/MiniBotFramework/Sensing/__init__.py

```diff
@@ -1,2 +1,4 @@
 import MiniBotFramework.Sensing.Sensor
+import MiniBotFramework.Sensing.ColorSensor
 import MiniBotFramework.Sensing.GPIOSensor
+import smbus
```

Comment:
It looks like you are just importing this when it is used- why bother importing it in Lib's __init__?